src/inner-slider.js

```diff
@@ -409,13 +409,14 @@ export class InnerSlider extends React.Component {
       this.slideHandler(targetSlide);
     }
   };
-  clickHandler = e => {
+  stopPropagationIfNotClickable = e => {
     if (this.clickable === false) {
       e.stopPropagation();
       e.preventDefault();
     }
     this.clickable = true;
   };
+  clickHandler = this.stopPropagationIfNotClickable;
```

Comment:
Is this method name change only for readability ?

Reply:
@bolatovumar

Reply:
@akiran this `stopPropagationIfNotClickable` is also used on line 443 below. If I just keep it named as `clickHandler` it would be weird calling a click handler directly from another method. This name change reflects more accurately what these lines of code do.

Reply:
Hi, Adding this.clickable = true in swipeEnd method might be a better solution since we are setting clickable to false in swipeMove method.

Calling stopPropagationIfNotClickable in swipeStart is not correct.

Reply:
@akiran ok, i will try to add it to `swipeEnd`. I think I tried it already but it didn't work for me. I will try it agian.

Reply:
@akiran ok, i tried putting `this.clickable = true` into `swipeEnd` and it does fix this bug on mobile, however, it introduces another bug: now swiping on desktop becomes impossible. As soon as you swipe on desktop it takes you to the link specified. This shouldn't happen.

Having `this.stopPropagationIfNotClickable(e);` in `swipeStart` fixes the issue on mobile and doesn't introduce a new one on desktop.